server/protocols/webadmin-cli/README.md

```diff
@@ -123,4 +151,171 @@ Show all users' name on the list.
 ### Update a user password
 Same as Create, but a user need to exist.
 
-If the user do not exist, then it will be created.
\ No newline at end of file
+If the user do not exist, then it will be created.
+
+## Manage Mailboxes
+
+### Create a mailbox
+Create a specific mailbox for a user:
+
+```
+{cli} mailbox create <username> <mailboxName>
+```
+
+Resource name username should be an existing user.
+
+Resource name mailboxName should not be empty, nor contain # & % * characters.
+
+To create nested mailboxes, for instance a work mailbox inside the INBOX mailbox, people should use the . separator. E.g: INBOX.work
+
+### Test a mailbox existence
+
+```
+{cli} mailbox exist <usernameToBeUsed> <mailboxNameToBeTested>
+```
+
+Resource name usernameToBeUsed should be an existing user 
+
+Resource name mailboxNameToBeTested should not be empty
+
+### Delete a mailbox and its children
+
+```
+{cli} mailbox delete <usernameToBeUsed> --mailbox <mailboxNameToBeDeleted>
+```
+
+Resource name usernameToBeUsed should be an existing user 
+
+Resource name mailboxNameToBeDeleted should not be empty
+
+### Delete all mailboxes of a user
+
+```
+{cli} mailbox delete <usernameToBeUsed>
+```
+
+Resource name usernameToBeUsed should be an existing user
+
+### Get mailboxes list
+
+```
+{cli} mailbox list <usernameToBeUsed>
+```
+
+Resource name usernameToBeUsed should be an existing user
+
+### Reindex a user mailboxes
+
+```
+{cli} mailbox reindex --username <usernameToBeUsed>
```

Comment:
why here do you have a `--username <username>` option while in other commands you have only `<username>`?

Reply:
(can we try to be me more coherent?)